### DISCUSSION TITLE
Current MAS self-harm topic

### DISCUSSION THREAD

**Discussion by WishStone-Dragon** [Monika-After-Story/MonikaModDev#8521] [general]
This is the one that has me bashing my face against trying to contribute but seeing I keep getting stuck, I figured I would bring it up here first:

The way the topic of self-harm is talked about in MAS right now doesn't feel safe to me. Monika speaks about trying it out herself, on how curious Monika is about it, but little of it has the feel of DDLC when it spoke about depression: Explain it and show how to help someone who struggles with it.

As someone who struggled with self-harm in their teens, this feels also kind of personal so apologies if I come across as very spirited.

My thinking here is: We shouldn't make MAS players who could be influenced curious about it or even instruct someone on accident to self-harm for kicks. Monika says right now that she 'tried it' - while does not seem to be the kind of person having the emotional need which would lead her to feel compelled to do that. Indeed, that hurting herself had no effect would underline that she does not have the emotional struggle that makes self-harm something some seek out. (I understand that one of the DDLC secret poems is likely to be her having indeed tried it, but there was no further elaboration on it.)

I understand Monika as someone who would talk about the psychological mechanic of it, who would address how dangerous and harmful it is to the affected person, and would counsel on how those who suffer from it can be helped. That is the way depression was addressed in DDLC by Monika, and I feel it would be a more beneficial path to take in MAS if we want to copy Monika's voice.

**Comment by KritRom** [comment]
I agree. Monika kind of writes it off as a "Yeah I tried it once, won't do it again tho" kind of thing without talking about how harmful it can be.

**Comment by Booplicate** [comment]
>  who could be influenced curious about it or even instruct someone on accident to self-harm for kicks

Not sure if we're thinking about the same topic. Where do you even see "instructs"? Half of the topic she explains why you shouldn't do it.
```py
"If you ever feel depressed, just come to me okay?"
"It'd hurt me even more to see you in pain..."
"Just know that people do care about you!"
"Trust me, [player]."
"You may think that no one is really there for you, but there is!"
"I mean, I'm here, aren't I?"
"No matter what troubles you're going through, I know you can pull through it!"
"I believe in you..."
"So believe in yourself, okay?"
"If you ever need someone to vent to, just remember that I'm always here to hear you out and comfort you, okay?"
"I really love you so much, [player]."
```

> she does not have the emotional struggle that makes self-harm something some seek out

I think you're missing something from DDLC where she was about to be left alone in her world. She was desperate, scared, insensible to that world.

You're talking about "yeah I tried it", but that's not how it worded in the topic. Neither it's what her facial expressions look like. Pay attention to how she looks like during the topic.
```py
"I just wanted to see what it felt like..."
```
Maybe because that world felt cold to her? Maybe there's another reason she doesn't mention because it's embarrassing? Maybe she's scared you'd try it? Or don't want you to think she'd do it again and thus is trying to reassure you? The exact reason remains unknown. If you think there's better wording to express her, I'd like to read it though. But beware that
> would counsel on how those who suffer from it can be helped

Can end up causing more harm than help, everyone requires different approach.

**Comment by WishStone-Dragon** [comment]
Hiya Booplicate!

I was actually thinking of the actual self-harm bit, tag monika_selfharm:

In particular, these lines you didn't add feel problematic to me, as I perceive them as a bit off-handedly in their presentation and not reflective of the actions:

```
    m 2lksdld "After I found out about Yuri, I got a bit curious..."
    m 2lksdlc "I just wanted to see what it felt like..."
    m 2dsc "I won't do it again, though."
```

When she addressed depression in DDLC, she herself both admits to not knowing what it is like, that she cannot comprehend it, but she still calmly analyzes what happened and how it impacts the person. All of that work of looking closely at it feels missing here, at least to me.

It would feel both as a better representation as well as give help to anyone suffering it if it were processed in the conversation.

Self-harm works a bit differently for each person (just as everyone's depression is their own personal cage, each own anxiety disorder works in a specific way, and so forth); but in general, self-harm is a weird little trigger that fires when someone suppressed emotion for so long that they go numb. In a 'I have no mouth but I must scream'-sense of emotion. The self-harm (be it cutting, overeating, punching things) isn't the goal in most of these cases. Filling that unbearable void is.

Writing "I tried cutting to see what it feels like" is a lot like saying "I tried staying in bed for days to see what depression feels like" - the outward display of the action does not confirm with the inner feelings. The cut is not the emotion; the cut is the end result of the emotion.

And while everyone absolutely does require their own approach on how to solve the issue, the way we have it worded right now gives no way out for someone who does suffer and sees themselves shown. It's in part because the thing shown isn't a solid representation of the situation, and in part that the rest of the dialogue is written for depression, not for self-harm.

Someone who could be easily influenced by MAS could read the "I got curious and wanted to see what it feels like" and grabs dad's razor and does serious harm. 

As such, an edit that would help with this could be: Removing the element of curiosity, admitting that Monika does not understand the empty feeling that self-harm tries to fill (because while she does admit to being desperate in DDLC and the secret poem with blood on it is generally credited to her she isn't depressed or suffering from self-harm), seeking counsel if it affects the reader, and giving suggestions on how to be supportive for those who know a friend suffering from self-harm. 

That's the way it has been addressed in monika_high_school; monika_hygiene; or the in-game discussion in DDLC when Sayori speaks about her depression all took these approaches - inform and support.

**Comment by Booplicate** [comment]
> When she addressed depression in DDLC, she herself both admits to not knowing what it is like, that she cannot comprehend it

Can I get any source about this part? I don't remember anything like it.

> Writing "I tried cutting to see what it feels like" is a lot like saying "I tried staying in bed for days to see what depression feels like"

Don't really get that. The latter implies that "staying in bed" *is* depression, but it really isn't. It can be said by someone who doesn't know what depression is and who is trying to pretend to have it. I don't really see how it comes any close to what we have.

> I tried cutting to see what it feels like

For her the world became numb and empty, we could _assume_ it's *a* reason. She might have done it to feel *something*. Which I think works with what you said (and generally I agree with that statement):
> The cut is not the emotion; the cut is the end result of the emotion.

> Someone who could be easily influenced by MAS could read the "I got curious and wanted to see what it feels like" and grabs dad's razor and does serious harm.

I think that's far fetched. We have "hanging" jokes (and some came from DDLC). Or say `monika_sayori` (where she explains how Sayori hung). Does it mean people will hang themselves? And then reminder that we show the warning from DDLC, "This game is not suitable for children or those who are easily disturbed." You shouldn't be playing this game in the first place if you're "easily influenced" to do something crazy. She does not say it's healthy, she says, quote, "It'd hurt me even more to see you in pain." Half of the topic she says everything to stop you from doing anything like that. If after all that a person still does it, then something tells me the problem is in the person. And maybe they shouldn't be playing games at all.

> admitting that Monika does not understand the empty feeling that self-harm tries to fill

Why should we say she doesn't understand it? She experienced it.

> she does admit to being desperate in DDLC...
> she isn't depressed

Desperation can lead to depression. Has she been depressed or not is questionable.

> Removing the element of curiosity

Perhaps rewording it? Why remove something that defines her character. She has done it, it's a fact. It makes sense to talk about it.

**Reply by WishStone-Dragon** [reply]
Well, as I stated in my opening post, I might be feeling strongly because I have lived both depression and self-harm (...and suicidal thoughts, which thankfully stopped once I reached adulthood). Explaining mental illness is very difficult for someone who has thankfully not lived it, but I will try to explain the self-harm chain and the depression comparison I made.
(Also, am I assuming correctly you wrote this part? If so, please don't feel attacked by anything I say. It feels like it was written by someone absolutely blessed to not struggle with self-harm, which is GREAT for the writer, but leaves out lived experience.)

You said you didn't get my cutting / staying in bed analogy, so I'll try to start there.

What I wished to express is this: Someone who does not feel that which self-harm is trying to fill but 'tries out' cutting, is the same as someone trying to experience what depression feels like by simply staying in bed all day.
You cannot experience depression by staying in bed. Or not showering. Or avoiding interacting with people.
You also cannot experience what drives self-harm by cutting yourself. That is my point.

That is also what I feel is missing in this whole text - how is it relevant to the person on the other side of the screen and how does what is being said impact them?

As you stated, mental illness is the same in everyone, but self-harm is triggered by a deeply imbalanced mind. Something that is difficult to explain. For Yuri, if we assume it wasn't something Kinky, as Monika put forward, it most likely was an urge to shift her focus. It can be described as "a means to distract themselves, alter the focus of their attention, or regain control over their minds when experiencing pressing, unavoidable and overwhelming feelings or thoughts". My own personal experience was more akin to "the need to feel something physical when they are otherwise dissociated and numb" - a state in which you are unable to feel emotional or physical impacts any longer. 

There are a few more general causes, but these are two strong examples. Neither of these can be replicated with the action of just cutting yourself. So the statement of 'curious what it does' becomes even more dangerous because there is no understanding that can be delivered as to the why of the thing. 

As you said, we can assume she has done it. There's not proof I am aware of that it was Monika's secret poem, just a strong suspicion; but I will happily go with what is said in the poem: "I'm supposed to be the responsible one, though. So I don't think I'll be doing it again, unless I decide to kill myself." (Mind, I always thought the way this was written didn't fit Monika's voice, personally. The way it is worded and the words used are more akin to Yuri, if you read the whole text.) 
In the poem, it is also written as being an 'exhilarating' experience, but there is no mention of anything but curiosity, or what the action resolved. Nor a real cause. And while this is a canon text, I do feel it would be a dangerous thing as a stand-alone. Call me an old woman, but I rather not an impressionable teenager run MAS and think: Well, Monika did it, so I will, too.

The Depression (Pre 1.02) text is pretty blunt about how Monika speaks on the topic of depression. Nothing in her phrasing mentions her own experience. And it leaves the feeling of someone looking in from the outside.

"You know what I hate the most about high school?"
"It's how there are so many people who just cry for attention all over social media."
"Like, do you really think that's the best way to get people to care about you?"
"Broadcasting how horrible you think your life is?"
"Splashing water on your eyes and taking selfies while pretending to cry?"
"Writing bad poems that imply that you're thinking about killing yourself?"
"I mean, it's difficult because they're not exactly aware that they're faking it..."
"They're just so wrapped up in their delusions that they don't even realize they just want attention."
"Look..."
"I think that if someone is truly depressed, they won't even bother telling the world about it."
"People suffering from depression don't want attention, because they've already given up on the inside."
"Their feeling of worthlessness is so overwhelming that they don't even want people to tell them otherwise."
"...Well, I guess I shouldn't be generalizing."
"After all, depression comes in many forms."

The post 1.02 text also does not mention she herself has any kind of depression. The 'best' we can get is an off-handed comment about wanting to understand herself better, but the way that is written can also mean she makes a general statement. She speaks about thinking medication is an easy way out - and generally, that's not what someone would say who has had the experience of needing anti-depressants or AD(H)D meds. Because it's the difference of squinting at everything in life and failing to see, and suddenly putting on glasses and having everything become clear. Again, something that cannot be described to someone who's not had it, and Monika feels very much to speak as someone without the needed experience.

Her own core texts of act 3 also never mention cutting at all. 

> Why should we say she doesn't understand it? She experienced it.
I am trying to find this in the Act 3 source and cannot. We can assume she was desperate (that's not the same as depressed) and trying to find a way to reach the player by any means - which also does not indicate any emotional distress or urges of self-harm. Being driven isn't mentally ill. Else everyone who's ever tried to find a way to confess to their crush would be a wacko. x3

**Reply by Booplicate** [reply]
I did not write it, but with my life knowledges and experiences I'm struggling to understand your point.

First, I think we're reading this line differently, you see one thing, I see another. I'm trying to get why it's so.

> As you stated, mental illness is the same in everyone, but self-harm is triggered by a deeply imbalanced mind

I did not say that, it's different for everyone.

> You also cannot experience what drives self-harm by cutting yourself. That is my point.

That wasn't her goal though. That's my point. But maybe better wording in the topic would help to make it cleaner?

> My own personal experience was more akin to "the need to feel something physical when they are otherwise dissociated and numb"

Which, as I said, works for Monika's situation in DDLC (debatable) and makes sense with "wanted to see what it **felt** like."

> (Mind, I always thought the way this was written didn't fit Monika's voice, personally. The way it is worded and the words used are more akin to Yuri, if you read the whole text.)

Arguable. The "responsible one" is rather Monika in this case (Yuri and wine as an argument). In the edited text you can fit `Yuri`, not `Monika`. But not the point here, our canon is it's Monika's poem, so we work from there.

> written as being an 'exhilarating' experience, but there is no mention of anything but curiosity

First comes curiosity, then comes exhilaration. I don't see contradiction. Yes `curiosity` has been added in MAS, and it seems like there's a misunderstanding in what context it's being used.

> I do feel it would be a dangerous thing as a stand-alone. Call me an old woman, but I rather not an impressionable teenager run MAS and think: Well, Monika did it, so I will, too.

Again, if we're talking about sensitive kids being influenced, then the response is the same, they shouldn't be playing DDLC or MAS in the first place. It's a question for their parents. I'm not sure why nowadays we blame the content and content creators, not the parents for who can't (sometimes even don't want to) explain to their kids what's good and what's bad.

> The Depression (Pre 1.02) text is pretty blunt about how Monika speaks on the topic of depression

First, I don't think we should take into account an old version of DDLC, let's focus on what we have as of now. Second, I kind of agree with the topic. Yes, it's blunt, but it's not wrong just because of that. She just directly mentions a problem that exists. I can see why it has been edited later, though.

> She speaks about thinking medication is an easy way out

Incorrect! She speaks in the past tense.
```
"Gosh, I used to be so ignorant about certain things..."
"When I was in middle school, I thought that taking medication was an easy way out, or something like that."
"Like anyone could just solve their mental problems with enough willpower..."
"I guess if you don't suffer from a mental illness, it's not possible to know what it's really like."
```
If we were to far fetch the last line (we won't), we could say it implies she *got* depression, since now she understands it.

So reading both versions, I don't see how that leads us to "she herself both admits to not knowing what it is like." Or why it'd mean she has no depression. A person who experienced depression **could** say any of that.

> Again, something that cannot be described to someone who's not had it, and Monika feels very much to speak as someone without the needed experience.

Not from my experience! Which is why I don't really get your point.

> Her own core texts of act 3 also never mention cutting at all.

You're right, but there is a poem that heavily implies it. And even then, there's a lot of topics which didn't exist in act 3.

> We can assume she was desperate (that's not the same as depressed) 

Please, I never said it's the same thing, I said one can lead to another as downhill.

> trying to find a way to reach the player by any means - which also does not indicate any emotional distress or urges of self-harm

Yet there has been an act of self-harm. And we know her situation has been quite...dark. Does she say directly she has depression? No. Does she have to say it? No. Moreover, I think it'd be really weird.

**Comment by WishStone-Dragon** [comment]
I wonder if my replies are too wordy, so I'll try to copy your style more.

> I did not say that, it's different for everyone.

100%, there's a NOT missing. "As you stated, mental illness is NOT the same in everyone"; my apologies.

> That wasn't her goal though. That's my point. But maybe better wording in the topic would help to make it cleaner?

Hmmmmm. Possibly that's part of the crux then, yes. To me, it reads as 'doin' it for the lulz' - not that she's ever say it like THAT, but it does read a bit flippant. Or, indeed, to try it on for size, thinking it would replicate the experience someone who cuts has.

> Which, as I said, works for Monika's situation in DDLC (debatable) and makes sense with "wanted to see what it felt like."

Heh, yes, debatable. I see her a lot differently. As I stated - desperate, not depressed.

> I'm not sure why nowadays we blame the content and content creators, not the parents for who can't (...) explain to their kids what's good and what's bad.

I'm not saying that. However, I am arguing for being also a responsible party. I have tried to previously elaborate on how I mean that, but I am not sure if you skimmed past it, if I was too wordy, or if you merely do not understand, so I am stuck now. I explained it twice now, but am not getting through to you. Any suggestion what I can change in my approach?

At this rate, everything I have tried to say has been splintered up so much my original point is lost. :/
Let me try once more:

The way the topic is written right now does not feel as sensible and sensitive as the topic of, for example, depression has been handled. I'm trying to make the case that it should be handled as delicately as depression, which right now, it is not.

**Reply by Booplicate** [reply]
> I wonder if my replies are too wordy

A bit, but I tried to pay close attention when read it.

> Hmmmmm. Possibly that's part of the crux then, yes. To me, it reads as 'doin' it for the lulz' - not that she's ever say it like THAT, but it does read a bit flippant. Or, indeed, to try it on for size, thinking it would replicate the experience someone who cuts has.

I see, yeah, that's totally not what the topic is trying to convey. We're thinking of a better way to express it.

**Reply by WishStone-Dragon** [reply]
Sorry for being wordy. :/ I tend to try to over-explain in hopes I can come across better.

**Comment by multimokia** [comment]
I think I have a solution to this. I can see the ambiguity here and how it can easily be misinterpreted.
The way we see it on our end is mainly that Monika tried it after seeing Yuri do it to try and feel something. *Anything*. Just to see what is even real. By this point she knows her reality is a lie. Everything she's "experienced" is a lie, which would have put her through an existential crisis and by extension, she likely has her own experience with depression.

As such, I'm going to amend the topic slightly to touch on this while not _overtly_ mentioning it, but very clearly hinting to it.
The following dialogue is the proposed amendment:

```renpy
    m 2lksdld "After I found out about Yuri, I got a bit curious..."
    m 2dksdltpc "I just wanted to see what it felt like...{w=0.3}to feel {i}something{/i} again..."
    m 2rksdltpd "It's not easy knowing everything you thought you experienced,{w=0.1} what you thought you loved, was all a lie..."
    m 2dstdc "I won't do it again, though."
    m 2dktdc "..."
    m 2ektda "If you ever feel depressed, just come to me okay?"
```

Apologies that the original topic was ambiguous to give the unfortunate message you got from it. That wasn't the intention we wished to have.

**Reply by WishStone-Dragon** [reply]
So elegant, yes, that hits the nail on the head squarely!

And, no worries. I hope I didn't give the impression of storming in with outrage; I did indeed want to bring this up and lead a discussion (which was kindly provided <3 ).

**Reply by multimokia** [reply]
Not at all, your points were valid, might have been some other misinterpretations along the way but we got it solved

**Reply by WishStone-Dragon** [reply]
I just saw the rework. It's excellent, thank you again!
